src/pages/dmt5/_dhero.js

```diff
@@ -0,0 +1,359 @@
+    }
+`
+
+const LinkWrapper = styled.div`
+    display: flex;
+    margin-top: 3.2rem;
+
+    @media (max-width: 1420px) {
+        top: 480px;
+    }
+    @media ${device.laptopM} {
+        top: 350px;
+    }
+    @media ${device.tabletL} {
+        top: 236px;
+    }
+    @media ${device.tablet} {
+        position: unset;
+        top: unset;
+        justify-content: start;
+    }
+    @media (max-width: 460px) {
+        flex-wrap: wrap;
+    }
+`
+
+const GoToLiveDemo = styled(Button)`
+    color: var(--color-white);
+    border-color: var(--color-black-5);
+    margin-left: 1.6rem;
+    padding: 1.1rem 1.6rem;
+    width: auto;
+
+    @media ${device.laptopM} {
+        font-size: 14px;
+        margin-left: 0;
+    }
+    @media ${device.mobileL} {
+        max-width: 100%;
+        padding: 10px 11px;
+        white-space: nowrap;
+    }
+`
+const DemoButton = styled(LinkButton)`
+    padding: 1.4rem 1.6rem;
+    width: auto;
+    font-size: 14px;
+    margin-right: 2px;
+    border: unset;
+
+    @media ${device.mobileL} {
+        padding: 1.5rem 1.6rem;
+        height: 42px;
+        white-space: nowrap;
+    }
+    @media (max-width: 460px) {
+        margin-bottom: 2rem;
+    }
+`
+const StyledContent = styled(Header)`
+    font-size: 5.6rem;
+
+    @media (max-width: 1322px) {
+        font-size: 4.2rem;
+    }
+`
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 562px;
+
+    @media (max-width: 1320px) {
+        max-width: 450px;
+    }
+    @media ${device.laptop} {
+        max-width: 390px;
+    }
+    @media ${device.laptopM} {
+        max-width: 410px;
+    }
+    @media ${device.tabletL} {
+        width: 46%;
+        max-width: 400px;
+        top: 60px;
+    }
+    @media ${device.tablet} {
+        width: 100%;
+        top: 350px;
+    }
+    @media ${device.mobileL} {
+        top: 280px;
+        max-width: 328px;
+    }
+    @media ${device.mobileM} {
+        top: 230px;
+        max-width: 328px;
+    }
+`
+
+const query = graphql`
+    query {
+        dbot: file(relativePath: { eq: "dbot_trade.png" }) {
+            ...fadeIn
+        }
+        dmt5: file(relativePath: { eq: "dmt5_trade.png" }) {
+            ...fadeIn
+        }
+        dtrader: file(relativePath: { eq: "dtrader_trade.png" }) {
+            ...fadeIn
+        }
+        dtrader_mobile: file(relativePath: { eq: "dtrader_trade_mobile.png" }) {
+            ...fadeIn
+        }
+        dbot_mobile: file(relativePath: { eq: "dbot_trade_mobile.png" }) {
+            ...fadeIn
+        }
+        dmt5_mobile: file(relativePath: { eq: "dmt5_trade_mobile.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const DHero = ({
+    title,
+    background_alt,
+    background_svg,
+    background_svg2,
+    content,
+    image_name,
+    is_mobile,
+    join_us_for_free,
+    go_to_live_demo,
+    Logo,
+}) => {
+    const data = useStaticQuery(query)
+    const handleRedirect = () => {
```

Comment:
{ 
      const path = image_name === 'dbot' ? '/bot' : '/' ;
       return deriv_app_url + path 
} 
then you can use it with `<LocalizedLink  external to={handleRedirect()}/>`
since this link is an `external` and we should show a confirmation for EU clients, pleaase use <LocalizedLink external/>.